### DISCUSSION TITLE
shortcut to select parent column

### DISCUSSION THREAD

**Discussion by age3rcm** [opentoonz/opentoonz#6819] [Ideas]
A shortcut to select parent/children colluumn linked in the schematic view is another much used thing for cut-out animation
it speeds up process to animate different parts

a shortcut to select parent/children of the elements
![vivaldi_1mhVCQ8GiH](https://user-images.githubusercontent.com/36406840/117212470-c5e73980-add0-11eb-8a5b-d0a019c292ee.png)
example tof toonboom

**Comment by DarrenTAnims** [comment]
The schematic is a great way to select parent/child columns. And you can also use the skeleton tool to select the drawing in any column for moving, which looking like the image you shared.

**Comment by age3rcm** [comment]
just think about a simple button to press instead of going in other menus or tools each time
also being able to click on any node in the chain to go to the desired element

**Comment by gab3d** [comment]
a simple shortcut is even more practical
in Maya you use the arrow keys to move up/down a hierarchy in the viewport (even left/right ones travels you through siblings).
